Java/HTML (JSP).sublime-syntax

```diff
@@ -12,6 +12,25 @@ file_extensions:
   - jsp
   - jspf
   - jspx
+  - jstl
+
+variables:
```

Comment:
Maybe it would make sense to add a comment line here with a link to:

<https://github.com/kkos/oniguruma/blob/master/doc/UNICODE_PROPERTIES>

for those unfamiliar with the usage of named properties...maybe not. Your call.

Reply:
If you mean `\p{L}` and `\p{N}` ... those variables are just copied from Java.sublime-syntax which has been using them a long time. It's not the task of these syntax files to teach noops the basics of regular expressions or any other stupid basics. If someone is curious about them a quick google/duckduck search is a proper solution. Same I did, when I saw them the very first time.

Reply:
yeah ok, nvm then